### PR TITLE
Fix `ToolDefaultPropertiesExportBlacklistTest#export_tool_configuration()` when building the plugin on Windows

### PR DESCRIPTION
**Pull request by casz**
### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!

- [x] Please describe what you did

- [x] Link to relevant GitHub issues or pull requests

- [x] Link to relevant [Jenkins JIRA issues](https://issues.jenkins-ci.org)

- [x] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### Description

Argh!

### CODE REVIEW

File: integrations/src/test/java/io/jenkins/plugins/casc/ToolDefaultPropertiesExportBlacklistTest.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ public void export_tool_configuration() throws Exception {
         ConfigurationContext context = new ConfigurationContext(registry);
         CNode yourAttribute = getToolRoot(context);
 
-        String exported = toYamlString(yourAttribute);
+        String exported = toYamlString(yourAttribute).replaceAll("git\\.exe", "git");
```
Comment (casz) [inline integrations/src/test/java/io/jenkins/plugins/casc/ToolDefaultPropertiesExportBlacklistTest.java]: @timja you evil person 🤣

Comment (timja) [reply integrations/src/test/java/io/jenkins/plugins/casc/ToolDefaultPropertiesExportBlacklistTest.java]: bahahaha, really should be fixed in git plugin somehow 😉 
annoying that it configures a default for you

Comment (jansohn) [reply integrations/src/test/java/io/jenkins/plugins/casc/ToolDefaultPropertiesExportBlacklistTest.java]: A simple `replace("git.exe", "git")` would have been enough here, no need for a regular expression ;)

Comment (casz) [reply integrations/src/test/java/io/jenkins/plugins/casc/ToolDefaultPropertiesExportBlacklistTest.java]: @jansohn feel free to send a PR 😅

**Pull request merged**